service-runner/src/main/resources/profiles/default/config.env

```diff
@@ -1,14 +1,14 @@
 # secrets
 secret.data.username=postgres
 secret.data.password=password
-secret.platform_api.auth_secret=myAnchorToPlatformSecret
-secret.callback_api.auth_secret=myPlatformToAnchorSecret
+secret.platform_api.auth_secret=myAnchorToPlatformSecret1234567890
```

Comment:
What is the purpose of the trailing `1234567890`?

Reply:
Make key seize 256 bit